src/parser/statement.js

```diff
@@ -844,7 +844,7 @@ pp.parseExport = function (node) {
     }
     node.declaration = expr;
     if (needsSemi) this.semicolon();
-    this.checkExport(node);
+    this.checkExport(node, true, true);
```

Comment:
@hzoo: Do you mean `checkExport(node: Node, options: { named: boolean, default: boolean })` or what do you mean with object? Maybe also  `checkExport(node: Node, type: "named" | "default" | null)`?